scriptmodules/ports/hexen2.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="hexen2"
+rp_module_desc="Hexen II - Hammer of Thyrion source port"
+rp_module_licence="GLP2 https://github.com/svn2github/uhexen2/blob/master/docs/COPYING"
```

Comment:
Please use this link (the actual contents) https://raw.githubusercontent.com/svn2github/uhexen2/master/docs/COPYING

Reply:
Also it should be GPL2 not GLP2 as mentioned before.

Reply:
Thank you. I overlooked it. It has now been changed as to your suggestion. :-)